static/js/zamboni/collections.js

```diff
@@ -640,4 +640,9 @@ $(document).ready(function () {
     if ($('#details-edit').length && $('div.notification-box').length) {
         $(document).scrollTop($("div.primary").position().top);
     }
+
+    var notificationBox = document.querySelector('.notification-box');
```

Comment:
If we need to change the DOM with JS on the action because we don't have a good element to target we should add a class somewhere we _can_ style instead of adding the margin/style directly.

That said, what is the selector for the white box? Surely we can add margin to it?